server/routes/tweets.js

```diff
@@ -5,57 +5,166 @@ const router = express.Router();
 const status = require("../config/status");
 const models = require("../models/index");
 
+const {
+  AgregaTweetsPorTema,
+  AgregaTweetsPorTemaEParlamentar,
+} = require("../utils/functions");
+
+const Tema = models.tema;
+const TemaProposicao = models.tema_proposicao;
 const Tweet = models.tweet;
+const TweetProposicao = models.tweet_proposicao;
+const Proposicao = models.proposicao;
 
 router.get("/parlamentares", (req, res) => {
-  Tweet.findAll({
-    group: ["id_parlamentar_parlametria"],
-    attributes: [
-      "id_parlamentar_parlametria",
-      [Sequelize.fn("COUNT", Sequelize.col("id_parlamentar_parlametria")), "atividade_twitter"]
-    ]
-  })
-  .then(tweets => res.status(status.SUCCESS).json(tweets))
-  .catch(err => res.status(status.BAD_REQUEST).json({ err }));
-});
-
-router.get("/parlamentares/:id_parlamentar", (req, res) => {
-  const id_parlamentar = req.params.id_parlamentar;
+  const tema = req.query.tema;
 
-  Tweet.findAll({
-    group: ["id_parlamentar_parlametria"],
-    attributes: [
-      "id_parlamentar_parlametria",
-      [Sequelize.fn("COUNT", Sequelize.col("id_parlamentar_parlametria")), "atividade_twitter"]
-    ]
-  }, {
-    raw: true
-  })
-  .then(tweets => {
-    const max = Math.max.apply(Math, tweets.map((tweet) => tweet.dataValues.atividade_twitter));
-    const min = Math.min.apply(Math, tweets.map((tweet) => tweet.dataValues.atividade_twitter));
-    Tweet.findOne({
+  if (tema === "undefined" || tema === "") {
```

Comment:
```suggestion
  if (typeof(tema) === "undefined" || tema === "") {
```
é preciso testar o tipo pra os casos que o tema não vem